webapps/frontend/lib/ovs/api.js

```diff
@@ -34,7 +34,7 @@ define([
 
         callData = {
             type: type,
-            timeout: generic.tryGet(options, 'timeout', 1000 * 30),
+            timeout: generic.tryGet(options, 'timeout', 1000 * 120),
```

Comment:
Alright. I'm awaiting a merge in the pyrakoon to be able to overrule the timeouts.